rest/v10/index.ts

```diff
@@ -1023,11 +1023,23 @@ export const CDNRoutes = {
 	 * - GET `/avatar-decorations/{user.id}/{user.avatar_decoration}.png`
 	 *
 	 * This route supports the extension: PNG
+	 *
+	 * @deprecated Use {@link CDNRoutes.avatarDecoration} instead.
```

Comment:
This doesn't appear to work with the current documentation https://discord-api-types-git-fork-sdanialraza-feat-su-9a6e74-discordjs.vercel.app/api/next/discord-api-types-v10#CDNRoutes

```suggestion
	 * @deprecated Use {@apilink CDNRoutes#avatarDecoration} instead.
```